bots/images/scripts/ovirt.setup

```diff
@@ -0,0 +1,151 @@
+#!/bin/bash
+
+set -eux
+
+echo foobar | passwd --stdin root
+
+/usr/bin/systemctl stop firewalld NetworkManager kdump
+/usr/bin/systemctl disable firewalld NetworkManager kdump
+iptables -F
+
+FQDN=ovirt.cockpit.lan
+hostnamectl set-hostname ${FQDN}
+
+# Latest stable repos
+rpm -ivh http://resources.ovirt.org/pub/yum-repo/ovirt-release41.rpm
+rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7
+rpm --import /etc/pki/rpm-gpg/RPM-GPG-ovirt-*
+rpm --import https://dl.fedoraproject.org/pub/epel/RPM-GPG-KEY-EPEL-7
```

Comment:
it is quite how level in fact, oVirt distributes the repo rpm which includes and enables all the required dependencies. Except for the GPG keys which need to be imported. 
I can add "--nogpgcheck" if you prefer

for reference, that ovirt-release41.rpm it currently enables following repos:
repo id                                  repo name                        status
base/7/x86_64                            CentOS-7 - Base                   9,363
centos-opstools-release/7/x86_64         CentOS-7 - OpsTools - release       424
centos-ovirt-common-testing/x86_64       CentOS-7 - oVirt common              98
centos-ovirt42-testing/x86_64            CentOS-7 - oVirt 4.2                 46
centos-qemu-ev-release/x86_64            CentOS-7 - QEMU EV                   82
extras/7/x86_64                          CentOS-7 - Extras                   449
opstools7-fluentd-012-testing/x86_64     CentOS-7 - OpsTools - fluentd te    118
opstools7-perfmon-common-testing/x86_64  CentOS-7 - OpsTools - perfmon te     66
opstools7-sensu-common-testing/x86_64    CentOS-7 - OpsTools - sensu test     77
ovirt-4.1/7                              Latest oVirt 4.1 Release          1,034
ovirt-4.1-centos-gluster38/x86_64        CentOS-7 - Gluster 3.8              226
*ovirt-4.1-epel/x86_64                   Extra Packages for Enterprise Li 11,934
ovirt-4.1-patternfly1-noarch-epel/x86_64 Copr repo for patternfly1 owned       2
ovirt-centos-ovirt41/7/x86_64            CentOS-7 - oVirt 4.1                427
ovirt-master-centos-gluster310/x86_64    CentOS-7 - Gluster 3.10              87
*ovirt-master-epel/x86_64                Extra Packages for Enterprise Li 11,934
ovirt-master-snapshot/7                  Latest oVirt master nightly snap    432
ovirt-master-snapshot-static/7           Latest oVirt master additional n    122
ovirt-master-virtio-win-latest           virtio-win builds roughly matchi     35
sac-gdeploy/x86_64                       Copr repo for gdeploy owned by s      2
updates/7/x86_64                         CentOS-7 - Updates                2,146
virtio-win-stable                        virtio-win builds roughly matchi      4